bundles/liberty/src/liberty/tools/DevModeOperations.java

```diff
@@ -868,23 +833,15 @@ private String getMavenCommand(String projectPath, String cmdArgs) {
             baseCmd = isWindows() ? "mvn.cmd" : "mvn";
         }
 
-        // 2. Check if an environment variable was defined to point to the Maven installation.
-        if (mvnCmd == null) {
-            String mvnInstallPath = getMavenInstallHome();
-            if (mvnInstallPath != null) {
-                mvnCmd = Paths.get(mvnInstallPath, "bin", baseCmd).toString();
-            }
-        }
-
-        // 3. Use the base command.
+        // 2. Use the base command.
```

Comment:
This all makes sense and is a good change.

It might be just a bit more readable to have line 833 as:
`mvnCmd = isWindows() ? "mvn.cmd" : "mvn";`

and delete lines 836-839.

(And the same for Gradle).

I